client/src/containers/EditTrip/index.js

```diff
@@ -32,8 +36,19 @@ class EditTrip extends PureComponent {
 
     if (updateResult) {
       nextProps.history.push(`${routes.TRIPS}/${trip._id}`);
-    } else {
+    } else if (trip) {
+      let editorState = EditorState.createEmpty();
+      try {
```

Comment:
I do not really like this try-catch, but this is the only solution that I have found to keep the raw state in the database...